prody/proteins/interactions.py

```diff
@@ -0,0 +1,2043 @@
+            
+            kw = {'occupancy': lista_ext}
+            if 'output' in kwargs:
+                writePDB(kwargs['output'], atoms, **kw)  
+                LOGGER.info('PDB file saved.')
+            else:
+                writePDB('output', atoms, **kw)
+                LOGGER.info('PDB file saved.')
+        except: LOGGER.info('There is a problem.')
+        
+
+    def getFrequentInteractions(self, contacts_min=3):
+        """Provide a list of residues with the most frequent interactions based on the following interactions:
+            (1) Hydrogen bonds (hb)
+            (2) Salt Bridges (sb)
+            (3) Repulsive Ionic Bonding (rb) 
+            (4) Pi stacking interactions (ps)
+            (5) Pi-cation interactions (pc)
+            (6) Hydrophobic interactions (hp)
+        
+        :arg contacts_min: Minimal number of contacts which residue may form with other residues. 
+        :type contacts_min: int, be default 3.  """
+
+        atoms = self._atoms   
+        interactions = self._interactions
+        
+        InteractionsMap = np.empty([atoms.select('name CA').numAtoms(),atoms.select('name CA').numAtoms()], dtype='S256')
+        resIDs = list(atoms.select('name CA').getResnums())
+        resChIDs = list(atoms.select('name CA').getChids())
+        resIDs_with_resChIDs = list(zip(resIDs, resChIDs))
+        interaction_type = ['hb','sb','rb','ps','pc','hp']
+
+        for nr,i in enumerate(interactions):
+            if i != []:
+                for ii in i: 
+                    m1 = resIDs_with_resChIDs.index((int(ii[0][3:]),ii[2]))
+                    m2 = resIDs_with_resChIDs.index((int(ii[3][3:]),ii[5]))
+                    InteractionsMap[m1][m2] = interaction_type[nr]+':'+ii[0]+ii[2]+'-'+ii[3]+ii[5]
+        InteractionsMap = InteractionsMap.astype(str)
+
+        ListOfInteractions = [ list(filter(None, InteractionsMap[:,j])) for j in range(len(interactions[0])) ]
+        ListOfInteractions = list(filter(lambda x : x != [], ListOfInteractions))
+        ListOfInteractions = [k for k in ListOfInteractions if len(k) >= contacts_min ]
+        ListOfInteractions_list = [ (i[0].split('-')[-1], [ j.split('-')[0] for j in i]) for i in ListOfInteractions ]
+        LOGGER.info('The most frequent interactions between:')
+        for res in ListOfInteractions_list:
+            LOGGER.info('{0}  <--->  {1}'.format(res[0], '  '.join(res[1])))
+
+        LOGGER.info('Legend: hb-hydrogen bond, sb-salt bridge, rb-repulsive ionic bond, ps-Pi stacking interaction,'
+                                                    ' pc-Cation-Pi interaction, hp-hydrophobic interaction')    
+        from toolz.curried import count
+        LOGGER.info('The biggest number of interactions: {}'.format(max(map(count, ListOfInteractions))))
+        
+        return ListOfInteractions_list
+        
+
+    def showFrequenctInteractions(self, cutoff=5, **kwargs):
```

Comment:
this function name has a typo. There shouldn't be a c before the t

Reply:
Done